app/main.js

```diff
@@ -21,8 +22,18 @@
         "URL_LAUNCHER_OVERLAY_SCROLLBARS": process.env.URL_LAUNCHER_CONSOLE == null ? 0 : process.env.URL_LAUNCHER_CONSOLE === '1'
     };
 
-    let window = null;
+    if (process.env.NODE_ENV === 'development') {
+      console.log('Running in development mode')
+      electronConfig = Object.assign(electronConfig, {
```

Comment:
`Object.assign` modifies the object in place, no reason to use its return value. It's good practice to use `const` anyway, so in https://github.com/resin-io/resin-electronjs/pull/32 I made `electronConfig` a constant.